### ISSUE TITLE
Add to Plugins Directory

### ISSUE THREAD

**Issue by aljawaid** [theobald-software/KanboardPermalink#2]
Hello, can this useful plugin be added to the Plugins Directory?

If it is not actively maintained, add me as to the repository and I dont mind updating it and submitting it to the directory.

**Comment by maxwowpow** [comment]
Hello. Could you summarize the process of submitting it to the directory? It's like npm and every version have to be pushed to the directory? Does it support direct pull from github or repo discovery for the current version?

**Comment by aljawaid** [comment]
Hi, first you will need to publish a release here... Then, if the plugin file structure is okay, you need to fork the kanboard's website and do a pr in the plugins files. https://github.com/kanboard/website 

If you want me to do it, just add me to this repository.

**Comment by aljawaid** [comment]
Hi you assigned this to me but I need to be able to publish a release in order to submit to the directory.... please review the project settings to let me publish a release

**Comment by aljawaid** [comment]
@maxwowpow Please can you change the repository name to...: `KanboardPermalink`  This will make things a lot easier in the plugin folder structure... so the link should be: https://github.com/theobald-software/KanboardPermalink/

**Comment by aljawaid** [comment]
Fixed by https://github.com/theobald-software/KanboardPermalink/commit/22e1d6d721256c0e80034bca5ea3e52d5377ad0b

**Comment by aljawaid** [comment]
https://kanboard.discourse.group/t/revised-plugin-kanboardpermalink/2452

**Comment by aljawaid** [comment]
![image](https://user-images.githubusercontent.com/10233708/202692160-a639d058-5075-4e34-93f4-e16238c78830.png)
